### ISSUE TITLE
Error during deployment to eclipse daily site 

### ISSUE THREAD

**Issue by iloveeclipse** [spotbugs/spotbugs#531]
Just after the merge of 3.1 to 4.0 branch, the build job failed to deploy Eclipse bits to the gh-pages.

I'm not sure if this is transient infrastracture issue or something broken in the build (may be due the upgrade to gradle 4.2.1?).

https://travis-ci.org/spotbugs/spotbugs/jobs/328396510
```
Deploying application
Initialized empty Git repository in /tmp/d20180113-6904-5hvzds/work/.git/
Switched to a new branch 'gh-pages'
cd -
cd /tmp/d20180113-6904-5hvzds/work
rsync: change_dir "/home/travis/build/spotbugs/spotbugs/home/travis/build/spotbugs/spotbugs/eclipsePlugin/build/site/eclipse-daily" failed: No such file or directory (2)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1183) [sender=3.1.0]
Could not copy /home/travis/build/spotbugs/spotbugs/home/travis/build/spotbugs/spotbugs/eclipsePlugin/build/site/eclipse-daily.
failed to deploy
```

**Comment by iloveeclipse** [comment]
Something similar to #310 ? See commit  27fb3c44b5107ff79fb98611c73d5d930924d7f5?

**Comment by iloveeclipse** [comment]
Merged #532 but still the error is there in https://travis-ci.org/spotbugs/spotbugs/jobs/328408699:
```
rsync: change_dir "/home/travis/build/spotbugs/spotbugs/home/travis/build/spotbugs/spotbugs/eclipsePlugin/build/site/eclipse-daily" failed: No such file or directory (2)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1183) [sender=3.1.0]
Could not copy /home/travis/build/spotbugs/spotbugs/home/travis/build/spotbugs/spotbugs/eclipsePlugin/build/site/eclipse-daily.
```
Just wondering: the used path `/home/travis/build/spotbugs/spotbugs/home/travis/build/spotbugs/spotbugs` looks like a duplicated concatenation of `/home/travis/build/spotbugs/spotbugs`.

The code uses `local_dir: $TRAVIS_BUILD_DIR/eclipsePlugin/build/site/eclipse-daily`. Do they threat this now as relative to `$TRAVIS_BUILD_DIR`, so that it is duplicated???

**Comment by iloveeclipse** [comment]
Fixed with #533 .
